Release Notes/Release Notes v2.9.0.md

```diff
@@ -1,3 +1,30 @@
 # .NET Driver Version 2.9.0 Release Notes
 
-This file is a placeholder.
\ No newline at end of file
+The main new features in 2.9.0 are:
+
+* Sharded transactions using the same API as replica set transactions
+* Sharded transactions pinning to a single mongos router
+* Convenient API for transactions
+* Support for message compression
+* SRV polling for mongodb+srv connection scheme
+* Retryable reads on by default
```

Comment:
It's confusing to use the same language for both retryable reads _and_ writes, since the former is a new feature and the latter is not.  Emphasize that the driver adds support for retying read operations, and that it's enabled by default.  And that retyable writes is now enabled by default.

Reply:
A good idea. Changes made!